<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLibraryEditor</class>
 <widget class="QWidget" name="FormLibraryEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library static data</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="listWidgetCatalog">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Competition type</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sport range</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Climbing difficulty</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Age groups</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Teams</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Competotors</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Users</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="pageCompetitonTypes">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableView" name="tableViewCompetitionTypes"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSportRanges">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListView" name="listViewSportRanges"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageClimbingDifficulty">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListView" name="listViewDiffucalties"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageAgeGroups">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableView" name="tableViewAgeGroups">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageTeams">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTableView" name="tableViewTeams">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageCompetitors">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QTableView" name="tableViewCompetitors"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageUsers">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableViewUsers">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidgetCatalog</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
